Jenkinsfile

```diff
@@ -0,0 +1,49 @@
+import groovy.json.JsonSlurper
+// This Jenkinsfile is used by Jenkins to run the DataExporter step of Reactome's release.
+
+import org.reactome.release.jenkins.utilities.Utilities
+
+// Shared library maintained at 'release-jenkins-utils' repository.
+def utils = new Utilities()
+
+pipeline {
+	agent any
+
+	stages {
+		// This stage builds the jar file using Maven.
+		stage('Setup: Build jar file'){
+			steps{
+				script{
+					utils.buildJarFile()
+				}
+			}
+		}
+
+		stage('Main: Run Data-Exporter'){
+			steps{
+				script{
+					dir("${env.ABS_RELEASE_PATH}/data-exporter/"){
```

Comment:
This path made more sense when it was calling Perl scripts. I'll probably remove this `dir`